app/src/test/java/ju/video/player/view/information/search/FolderPanelTest.java

```diff
@@ -0,0 +1,16 @@
+package ju.video.player.view.information.search;
```

Comment:
Add copy right to all the clases of testing please and to each classes your firm and describe briefly each class.